.github/workflows/src/context.js

```diff
@@ -215,6 +221,14 @@ export async function extractInputs(github, context, core) {
           }
         }
       }
+      if (!issue_number) {
```

Comment:
as discussed, you should be able to remove this, if we update all the workflows you depend on, to publish the "issue_number" artifact